README.md

```diff
@@ -19,21 +16,36 @@ In this workshop we'll be building on what we learnt in the [pg walkthrough](htt
 - Create a superuser with a password by typing `CREATE USER [the new username] WITH SUPERUSER PASSWORD '[the password of the database]';` (the password needs to be in quotes, otherwise you get an error).
 - Change ownership of the database to the new user by typing `ALTER DATABASE [name of the database] OWNER TO [the new username];`
 - Add a `config.env` file and add the database's url in this format:
-`DB_URL = postgres://[username]:[password]@localhost:5432/[database]`.
+`DB_URL = postgres://[username]:[password]@localhost:5432/[database]`. The database name needs to be in lower case.
 
```

Comment:
Interesting! Didn't know you could do this https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNSTRING

But of all [the options](https://stackoverflow.com/questions/9736085/run-a-postgresql-sql-file-using-command-line-arguments), I wonder whether we aren't better off pointing students to
> Use "trust authentication" for that specific user

within the setup phase